docs/math-support.md

```diff
@@ -0,0 +1,27 @@
+# Math Support
+
+The published pages don't support math formulas by default. To enable this feature, you can add the following code snippet to the end of `_layouts/page.html`:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/mathjax@2/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+<script type="text/x-mathjax-config">
+    MathJax.Hub.Config({
+        tex2jax: {
+            skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
+            inlineMath: [['$','$']]
+        }
+    });
+</script>
+```
+
+ If you want the index page of your Foam site to render maths, you'll need to add that to `_layouts/home.html` as well, or change the layout of the index page to be "page" instead of "home" by putting this Front Matter on the top of your `readme.md/index.md`:
+
+```
+---
+layout: home
```

Comment:
you mean `page`?

Reply:
yes, please see the commit below https://github.com/foambubble/foam/pull/109/commits/e0cf333c591abacbb11d856f4ccb012d74de6c9a